src/components/Votes/VoteDisplay/VoteDisplay.scss

```diff
@@ -108,11 +98,17 @@ $vote-display__progress-bar-color--depleted-dark: #4ae89f;
   display: flex;
   justify-content: center;
   align-items: center;
+  transition: background-color 150ms linear;
+
+  &:hover {
+    background-color: color.scale($color-lighter-gray, $lightness: -10%);
```

Comment:
please define it just once.
a sass function is sufficient to handle the changing colors